sql/moz-fx-data-shared-prod/fenix_derived/feature_usage_metrics_v1/query.sql

```diff
@@ -0,0 +1,462 @@
+-- Query for fenix_derived.feature_usage_metrics_v1
```

Comment:
Remove this 3 line comment, it does not provide any additional information to the reader of this query.